src/components/Site/Site.jsx

```diff
@@ -5,7 +5,7 @@ import { hot as Hot } from 'react-hot-loader';
 import DocumentTitle from 'react-document-title';
 
 // Import Utilities
-import { ExtractPages, ExtractSections, GetPageTitle } from '../../utilities/content-utils';
+import { extractPages, extractSections, getPageTitle } from '../../utilities/content-utils';
```

Comment:
Why all these name changes?

Reply:
I think that by convention methods that are capitalized in JavaScript are constructors, which is not the case for all these helper functions.

Reply:
Yeah, I think that's definitely at least the convention in node. I got in the habit of capitalizing all `import`s a while back to delineate them from local variables but it definitely doesn't seem to be a standard practice.